README.md

```diff
@@ -14,71 +16,252 @@ Takeaway Challenge
+```
+From the users perspective in IRB:
+
+```
+order = Order.new
+=> #<Order:0x00007f863783fda0 @menu={:fish=>5, :chips=>3, :salad=>2}> 
+order.menu
+=> {:fish=>"£5", :chips=>"£3", :salad=>"£2"} 
+```
 
+## User Story 2
+
+```
 As a customer
 So that I can order the meal I want
 I would like to be able to select some number of several available dishes
+```
+From the users perspective in IRB:
+```
+order = Order.new
+=> #<Order:0x00007f863783fda0 @menu={:fish=>5, :chips=>3, :salad=>2}> 
+order.update("1 fish, 3 salad, 2 chips")
+=> {:fish=>1, :salad=>3, :chips=>2}
+```
+
+## User Story 3
 
+```
 As a customer
 So that I can verify that my order is correct
 I would like to check that the total I have been given matches the sum of the various dishes in my order
+```
+From the users perspective in IRB:
+```
+order = Order.new
+=> #<Order:0x00007f863783fda0 @menu={:fish=>5, :chips=>3, :salad=>2}, @cost=0, @items=[]> 
+order.update("1 fish, 3 salad")
+=> {:fish=>1, :salad=>3}
+order.items
+=> {:fish=>1, :salad=>3} 
+order.cost
+=> 11
+```
+
+## User Story 4
 
+```
 As a customer
 So that I am reassured that my order will be delivered on time
 I would like to receive a text such as "Thank you! Your order was placed and will be delivered before 18:52" after I have ordered
 ```
+From the users perspective in IRB:
+```
+order = Order.new
+=> #<Order:0x00007f863783fda0 @menu={:fish=>5, :chips=>3, :salad=>2}, @cost=0, @items=[]> 
+order.update("1 fish, 3 salad")
+=> {:fish=>1, :salad=>3}
+order.cost
+=> 11
+order.confirm
+=> "Thank you! Your order was placed and will be delivered before 2020-03-29 18:22:59 +0100."
+```
 
-* Hints on functionality to implement:
-  * Ensure you have a list of dishes with prices
-  * Place the order by giving the list of dishes, their quantities and a number that should be the exact total. If the sum is not correct the method should raise an error, otherwise the customer is sent a text saying that the order was placed successfully and that it will be delivered 1 hour from now, e.g. "Thank you! Your order was placed and will be delivered before 18:52".
-  * The text sending functionality should be implemented using Twilio API. You'll need to register for it. It’s free.
-  * Use the twilio-ruby gem to access the API
-  * Use the Gemfile to manage your gems
-  * Make sure that your Takeaway is thoroughly tested and that you use mocks and/or stubs, as necessary to not to send texts when your tests are run
-  * However, if your Takeaway is loaded into IRB and the order is placed, the text should actually be sent
-  * Note that you can only send texts in the same country as you have your account. I.e. if you have a UK account you can only send to UK numbers.
+## My own Story
 
-* Advanced! (have a go if you're feeling adventurous):
-  * Implement the ability to place orders via text message.
+```
+As a customer
+So that I can change my order if it is wrong
+I would to be able alter my order before I confirm
```

Comment:
This is really cool!

So this answers my question above - I wonder if there's a clearer name than `update`? to demonstrate this behaviour - I can't think of a good one, but something like `replace`?

 I also wonder if a user might want to be able to just add one order of chips to an existing order without re-adding all of the other items again?